samples/contrib/aws-samples/README.md

```diff
@@ -120,64 +120,16 @@ There are two ways you can give them access to SageMaker.
 
 ### Sample MNIST dataset
 
-Use the following python script to copy train_data, test_data, and valid_data to your bucket.  
-[Create a bucket](https://docs.aws.amazon.com/AmazonS3/latest/gsg/CreatingABucket.html) in `us-east-1` region if you don't have one already. 
-For the purposes of this demonstration, all resources will be created in the us-east-1 region.
-
-
-Create a new file named s3_sample_data_creator.py with following content :
-```
-import pickle, gzip, numpy, urllib.request, json
-from urllib.parse import urlparse
-
-###################################################################
-# This is the only thing that you need to change to run this code 
-# Give the name of your S3 bucket 
-bucket = '<bucket-name>' 
-
-# If you are gonna use the default values of the pipeline then 
-# give a bucket name which is in us-east-1 region 
-###################################################################
-
-
-# Load the dataset
-urllib.request.urlretrieve("http://deeplearning.net/data/mnist/mnist.pkl.gz", "mnist.pkl.gz")
-with gzip.open('mnist.pkl.gz', 'rb') as f:
-    train_set, valid_set, test_set = pickle.load(f, encoding='latin1')
+The following commands will copy the data extraction pre-processing script to an S3 bucket which we will use to store artifacts for the pipeline.
 
+1. [Create a bucket](https://docs.aws.amazon.com/AmazonS3/latest/gsg/CreatingABucket.html) in `us-east-1` region if you don't have one already. 
+For the purposes of this demonstration, all resources will be created in the us-east-1 region.
 
-# Upload dataset to S3
```

Comment:
why remove this ?
How to get sample data for other pipelines ?

Reply:
Is it just the simple training pipeline that also uses this? I also saw there's a pipeline with just HPO that seemed to pop up without documentation, too.

Reply:
ya simple train pipeline and HPO both use it 
HPO has documentation ... basically the same readme that we have for mnist classification pipeline 
It mentions the HPO pipeline in the beginning 

https://github.com/kubeflow/pipelines/blob/master/samples/contrib/aws-samples/mnist-kmeans-sagemaker/README.md